appendix/session.qmd

```diff
@@ -0,0 +1,22 @@
+---
+title: Session Info
+subtitle: Session Info for reproducibility
+---
+
+------------------------------------------------------------------------
+
+## Session Info
+
+```{r sessioninfo, message=FALSE}
+#| code-fold: show
+
+sessionInfo()
+```
+
+## renv `.lock` file
+
+```{r snapshot, echo=FALSE, message=FALSE}
+renv::snapshot("..", lockfile = "renv.lock", prompt = FALSE, force = TRUE)
```

Comment:
create snapshot on file render - this could lengthen the process of book render a little but I feel it's more correct as it would snapshot truly used pkgs with its version